### PR TITLE
Updated fr_FR translation and updated main.js for new translations strings.

### PR DESCRIPTION
**Pull request by eraffaelli**
Updated fr_FR translation and updated main.js for new translation strings.

'Set {0} color' doesn't include translation and use 'background' and 'foreground' for all languages, so I replaced it with two new strings for the title :
- 'Set foreground color'
- 'Set background color'

### CODE REVIEW

File: src/packages/default/Draw/main.js [hunk 1]
```diff
@@ -391,9 +391,15 @@
...
   ApplicationDrawWindow.prototype.openColorDialog = function(param) {
     var self = this;
-
+    
+    var colorParam = null;
+    if(param === 'background')
+        colorParam = doTranslate("Set background color");
+    else if(param === 'foreground')
+        colorParam = doTranslate("Set foreground color");
+
```
Comment (andersevenrud) [inline src/packages/default/Draw/main.js]: This style is not allowed (and will not allow merging). Use curly-braces :)

**Comment by andersevenrud** [comment]
- https://github.com/os-js/OS.js/wiki/Style-guide
- https://github.com/os-js/OS.js/blob/master/CONTRIBUTING.md#making-and-submitting-changes

**Comment by eraffaelli** [comment]
Sorry, I forgot to add them (I hate when I don't have them ><)

**Comment by andersevenrud** [comment]
You also have some other style issues:

- 2 spaces (not tabs)
- Aways spaces when using parentheses

**Comment by andersevenrud** [comment]
Also:

- Always use single-quotes!

**Comment by andersevenrud** [comment]
You can see it in defail in the build log: https://travis-ci.org/os-js/OS.js/builds/104646538

**Comment by eraffaelli** [comment]
I'm on it.
I don't use space but mine haves 4 spaces.
I correct and I commit after.

**Pull request closed without merging**